### PR TITLE
island-ui/ Cards refactoring

### PR DESCRIPTION
**Pull request by DavidBachmann**
# Cards refactoring

- [x] Create `TopicCard` which replaces `LinkCard`
- [x] Create `ActionCard`
- [x] Deprecate `LinkCard`
- [x] Create `ProgressMeter` component which `ActionCard` needs

## What

-

## Why

-

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

File: libs/island-ui/core/src/lib/ProgressMeter/ProgressMeter.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  },
+  red: {
+    outer: 'red100',
+    inner: 'red400',
+  },
+  rose: {
+    outer: 'roseTinted100',
+    inner: 'roseTinted400',
+  },
+} as const
```
Comment (jeremybarbet) [inline libs/island-ui/core/src/lib/ProgressMeter/ProgressMeter.tsx:30]: is `as const` necessary here?

File: libs/island-ui/core/src/lib/TopicCard/TopicCard.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          >
+            {tag}
+          </Tag>
+        </Box>
+      )}
+    </FocusableBox>
+  )
+}
+
+// [1] Used like this to get Storybook props working. Will be fixed.
```
Comment (jeremybarbet) [inline libs/island-ui/core/src/lib/TopicCard/TopicCard.tsx:69]: You can remove this comment, already added a global comment on the readme about it

**Review by jeremybarbet — CHANGES_REQUESTED** [review]
Just some minor comments

File: libs/island-ui/core/src/lib/ProgressMeter/ProgressMeter.tsx (hunk 1 above)
Comment (DavidBachmann) [reply libs/island-ui/core/src/lib/ProgressMeter/ProgressMeter.tsx:30]: Yeah, else it's just strings.
`Type 'string' is not assignable to type '"transparent" | "currentColor" | "white" | "dotted"`

**Comment by DavidBachmann** [comment]
> Just some minor comments

Thanks. Addressed.

**Review by jeremybarbet — APPROVED** [review]
LGTM 🚀

**Pull request merged**